File: 2984/Assuntos Pendentes.py
stack = []


s = input()

for x in range(0, len(s)):
    if s[x] == "(":
        stack.append("(")
    else:
        if len(stack)>0:
            stack.pop()

if len(stack) == 0:
    print("Partiu RU!")
else:
    print("Ainda temos %s assunto(s) pendente(s)!"%len(stack))

File: 2984/Assuntos Pendentes.cpp
using namespace std;
#include <iostream>
#include <vector>
#include<bits/stdc++.h>
#include <algorithm>
#include <string>
#include <map>

int main(){

    
    stack <char> s;
    
    string a;
    
    getline(cin, a);
    
    for (int i = 0; i < a.size(); i++){
        if (a[i] == '('){
            s.push('(');
        }else{
            if (s.size() > 0){
                s.pop();
            }
        }
    }
    
    
    if( s.size() == 0)
        cout << "Partiu RU!" << endl;
    else
        cout << "Ainda temos "<<s.size()<<" assunto(s) pendente(s)!"<<endl;
}
